Tasks/MobileCenterDistribute/mobilecenterdistribute.ts

```diff
@@ -406,7 +406,7 @@ async function run() {
         let app = utils.resolveSinglePath(appFilePattern);
         tl.checkPath(app, "Binary file");
 
-        let continueIfSymbolsNotFoundVariable = tl.getVariable('VSMobileCenterUpload.ContinueIfSymbolsNotFound');
+        let continueIfSymbolsNotFoundVariable = tl.getVariable('MobileCenterDistribute.ContinueIfSymbolsNotFound');
```

Comment:
This variable is used by Mobile Center, we should not change it. It will break existing definitions created by Mobile Center.